.changeset/clever-hornets-talk.md

```diff
@@ -0,0 +1,7 @@
+---
+"blitz": patch
+"@blitzjs/next": patch
+"@blitzjs/generator": patch
+---
+
+Downgrade tslog to `4.7.4`
```

Comment:
Hey @justinsmid could you mention the reason for the downgrade with the link to the tslog issue here?

Reply:
Updated - let me know if that explanation suffices or if it should be more/less specific/elaborate.